WordPress/Classes/ViewRelated/Blog/RelatedPostsPreviewTableViewCell.m

```diff
@@ -0,0 +1,166 @@
+
+@property (nonatomic, strong) UILabel *headerLabel;
+@property (nonatomic, strong) NSArray *previewPosts;
+
+@end;
+
+@implementation RelatedPostsPreviewTableViewCell
+
+- (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
+{
+    self = [super initWithStyle:UITableViewCellStyleDefault reuseIdentifier:reuseIdentifier];
+    if (self) {
+        _enabledHeader = YES;
+        _enabledImages = YES;
+        _headerLabel = [[UILabel alloc] initWithFrame:CGRectZero];
+        _headerLabel.text = NSLocalizedString(@"Related Posts", @"Label for Related Post header preview");
+        _headerLabel.textColor = [WPStyleGuide greyDarken20];
+        _headerLabel.font = [WPFontManager openSansSemiBoldFontOfSize:11.0];
+        [self.contentView addSubview:_headerLabel];
+        
+        RelatedPostsPreview *preview1 = [[RelatedPostsPreview alloc] initWithTitle:NSLocalizedString(@"Big iPhone/iPad Update Now Available", @"Text for related post cell preview")
+                                                                              site:NSLocalizedString(@"in \"Mobile\"", @"Text for related post cell preview")
+                                                                         imageName:@"relatedPostsPreview1"];
+        RelatedPostsPreview *preview2 = [[RelatedPostsPreview alloc] initWithTitle:NSLocalizedString(@"The WordPress for Android App Gets a Big Facelift", @"Text for related post cell preview")
+                                                                              site:NSLocalizedString(@"in \"Apps\"", @"Text for related post cell preview")
+                                                                         imageName:@"relatedPostsPreview2"];
+        RelatedPostsPreview *preview3 = [[RelatedPostsPreview alloc] initWithTitle:NSLocalizedString(@"Upgrade Focus: VideoPress For Weddings", @"Text for related post cell preview")
+                                                                              site:NSLocalizedString(@"in \"Upgrade\"", @"Text for related post cell preview")
+                                                                         imageName:@"relatedPostsPreview3"];
+
+        _previewPosts = @[preview1, preview2, preview3];
+        
+        for (RelatedPostsPreview *relatedPostPreview in _previewPosts) {
+            relatedPostPreview.imageView = [[self class ] imageViewWithImageNamed:relatedPostPreview.imageName];
+            [self.contentView addSubview:relatedPostPreview.imageView];
+            relatedPostPreview.titleLabel = [[self class] titleLabelWithText:relatedPostPreview.title];
+            [self.contentView addSubview:relatedPostPreview.titleLabel];
+            relatedPostPreview.siteLabel = [[self class] siteLabelWithText:relatedPostPreview.site];
+            [self.contentView addSubview:relatedPostPreview.siteLabel];
+        }
+    }
+    return self;
+}
+
++ (UILabel *)titleLabelWithText:(NSString *)text
```

Comment:
(Suggestion): If this method is used to create a `UILabel` instance, that's gonna be maintained by `RelatedPostsPreview`, maybe this method should live within `RelatedPostsPreview` perhaps?